Classes/UT3PaladinProjectile.uc

```diff
@@ -6,6 +6,25 @@
 //-----------------------------------------------------------
 class UT3PaladinProjectile extends ONSShockTankProjectile;
 
+function SuperExplosion()
+{
+        local actor HitActor;
+        local vector HitLocation, HitNormal;
+
+        HurtRadius(ComboDamage, ComboRadius, class'UT3DmgType_PaladinShockBall', ComboMomentumTransfer, Location );
```

Comment:
You can also add a comment here saying that this is the only line that is different from the original.